ui/src/components/ServiceOfferingsView.tsx

```diff
@@ -93,9 +95,11 @@ function ServiceOfferingsView(props: any) {
             // @ts-ignore
             return (
                 <ui5.Card
+                    selection-mode="Single"
                     key={index}
                     onClick={() => {
-                        setPortal(createPortal(<ServiceOfferingsDetailsView secret={secret} offering={offering} />, document.getElementById("App")!!, window.crypto.randomUUID()))
```

Comment:
createPortal was removed but is still imported. setPortal and portal i no more needed.